main.go

```diff
@@ -11,21 +11,19 @@ import (
 )
 
 func main() {
-	println("Server is starting...")
+	log.Print("Server is starting...")
 
-	pokemonService, _ := service_csv.NewPokemonService(parameters.CsvPath)
+	var err error
 
+	pokemonService, err := service_csv.NewPokemonService(parameters.CsvPath)
```

Comment:
OK, you are actually using `err` from this method, what if there's an error here?

You need to handle the error! If there's an error, halt program execution, if not, continue.